library/perl

```diff
@@ -1,101 +1,133 @@
 Directory: 5.034.001-slim-bullseye
 
 Tags: 5.34.1-slim-buster, 5.34-slim-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.034.001-slim-buster
 
 Tags: 5.34.1-threaded, 5.34-threaded, 5.34.1-threaded-bullseye, 5.34-threaded-bullseye
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 Directory: 5.034.001-main,threaded-bullseye
 
 Tags: 5.34.1-threaded-buster, 5.34-threaded-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.034.001-main,threaded-buster
 
 Tags: 5.34.1-slim-threaded, 5.34-slim-threaded, 5.34.1-slim-threaded-bullseye, 5.34-slim-threaded-bullseye
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 Directory: 5.034.001-slim,threaded-bullseye
 
 Tags: 5.34.1-slim-threaded-buster, 5.34-slim-threaded-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.034.001-slim,threaded-buster
 
 Tags: 5.32.1, 5.32, 5.32.1-bullseye, 5.32-bullseye
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 Directory: 5.032.001-main-bullseye
 
 Tags: 5.32.1-buster, 5.32-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.032.001-main-buster
 
 Tags: 5.32.1-slim, 5.32-slim, 5.32.1-slim-bullseye, 5.32-slim-bullseye
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 Directory: 5.032.001-slim-bullseye
 
 Tags: 5.32.1-slim-buster, 5.32-slim-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.032.001-slim-buster
 
 Tags: 5.32.1-threaded, 5.32-threaded, 5.32.1-threaded-bullseye, 5.32-threaded-bullseye
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 Directory: 5.032.001-main,threaded-bullseye
 
 Tags: 5.32.1-threaded-buster, 5.32-threaded-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.032.001-main,threaded-buster
 
 Tags: 5.32.1-slim-threaded, 5.32-slim-threaded, 5.32.1-slim-threaded-bullseye, 5.32-slim-threaded-bullseye
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
 Directory: 5.032.001-slim,threaded-bullseye
 
 Tags: 5.32.1-slim-threaded-buster, 5.32-slim-threaded-buster
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+Architectures: amd64, arm32v7, arm64v8, i386
 Directory: 5.032.001-slim,threaded-buster
+
+Tags: 5.37.8, 5.37, blead, devel, 5.37.8-bullseye, 5.37-bullseye, blead-bullseye, devel-bullseye
```

Comment:
Wait, isn't `blead` the name used for the "master" / "main" development branch of Perl?  Is it possible this will make folks think this is an actual development build as opposed to the periodic development releases?  (Sorry, we're reaching the limits of my knowledge of how folks know/understand Perl versions and labels on them, even as a user of it myself. :sweat_smile:)

Reply:
Yeah, looking at https://dev.perl.org/perl5/source.html and https://www.nntp.perl.org/group/perl.perl5.porters/2023/01/msg265547.html, it appears these _are_ commonly referred to as "development releases" but _not_ necessarily as "blead releases" right?

> The master branch, where the development takes places, is named _blead_.

vs

> We are happy to announce version 5.37.8, the ninth development release of version 37 of Perl 5.

Reply:
This does seem wrong to me ... though I too am at my limit of process here. Maybe @haarg or @leonerd would know better (just two people off the top of my head)?

Reply:
Yeah, digging deeper down we actually have tools like plenv/perl-build take `blead` tarballs from the git archive cf https://github.com/tokuhirom/Perl-Build/blob/b5e809342bed7b8ba51b53210cf5e1665953b34c/script/perl-build#L70

We can take off the `blead` tag for now and use `devel` only.

Reply:
If it has a `5.x.y` version number then it isn't blead, it's a release. Depending on the parity of `x`, odd numbers are development releases, and even numbers are production releases. `blead` is the name of the main development git branch, which is always in flight.